btcpay-setup.sh

```diff
@@ -383,25 +448,33 @@ stop on runlevel [!2345]
 # respawn # might cause over charge
 
 script
-    . /etc/profile.d/btcpay-env.sh
+    . $BASH_PROFILE_SCRIPT
     cd \"\$BTCPAY_BASE_DIRECTORY/btcpayserver-docker\"
     . helpers.sh
     btcpay_up
 end script" > /etc/init/start_containers.conf
     echo -e "BTCPay Server upstart configured in /etc/init/start_containers.conf\n"
-if $START; then
-    initctl reload-configuration
-    echo "BTCPay Server started"
-fi
+
+	if $START; then
+		initctl reload-configuration
+		echo "BTCPay Server started"
+	fi
 fi
 
+# else if [ "$OSTYPE" == "darwin"* ]; then
+	# Mac OS
+	# TODO create an auto-start script on boot. Not sure if we really need this as docker can start on it's own? Maybe we can use Mac's launchd for this, but not sure...
+#fi
+
 cd "$(dirname $BTCPAY_ENV_FILE)"
 
-if [ ! -z "$OLD_BTCPAY_DOCKER_COMPOSE" ] && [ "$OLD_BTCPAY_DOCKER_COMPOSE" != "$BTCPAY_DOCKER_COMPOSE" ]; then
+if [[ ! -z "$OLD_BTCPAY_DOCKER_COMPOSE" ]] && [[ "$OLD_BTCPAY_DOCKER_COMPOSE" != "$BTCPAY_DOCKER_COMPOSE" ]]; then
     echo "Closing old docker-compose at $OLD_BTCPAY_DOCKER_COMPOSE..."
     docker-compose -f "$OLD_BTCPAY_DOCKER_COMPOSE" down -t "${COMPOSE_HTTP_TIMEOUT:-180}"
 fi
 
+return
```

Comment:
remove this

Reply:
Must've left this during testing. Now gone.